manifests/init.pp

```diff
@@ -118,8 +109,27 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0640',
-    content => "OPTIONS=\"-A -u ntp:ntp -p /var/run/ntpd.pid\"
-SYNC_HWCLOCK=yes\n",
+    content => "OPTIONS=\"-A -u ntp:ntp -p /var/run/ntpd.pid\"\n",
+    notify  => Service['ntpd']
+  }
+  file { '/etc/sysconfig/ntpdate':
+    ensure  => 'file',
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0640',
+    content => "SYNC_HWCLOCK=yes\n",
+    notify  => Service['ntpd']
```

Comment:
why the notify?

Reply:
I forgot to remove it after I had learned what `ntpdate` did